pkgs/applications/science/math/tilp2/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv
+, fetchurl
+, fetchpatch
+, autoreconfHook
+, pkgconfig
+, intltool
+, glib
+, gnome2
+, gfm
+, libticables2
+, libticalcs2
+, libticonv
+, libtifiles2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tilp2";
+  version = "1.18";
+  src = fetchurl {
+    url = "mirror://sourceforge/tilp/${pname}-${version}.tar.bz2";
+    sha256 = "0isf73bjwk06baz2gm3vpdh600gqck9ca4aqxzb089dmxriv6fkv";
+  };
+
+  patches = fetchpatch {
+    name = "remove-broken-kde-support.patch";
+    url = "https://aur.archlinux.org/cgit/aur.git/plain/remove-broken-kde-support.patch?h=tilp";
+    sha256 = "1fn6vh7r45spkwpmkvffkbn7zrcsdrs5mjmspd5rwi3jc12cy3ny";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkgconfig
+    intltool
+  ];
+
+  buildInputs = [
+    glib
+    gnome2.gtk
```

Comment:
does this possibly need wrapGAppsHook (does this have an UI and does that ui work?)

Reply:
It has an UI, and it does work, so I guess it doesn't need wrapGAppsHook